ui/apps/platform/src/utils/searchUtils.ts

```diff
@@ -242,6 +253,12 @@ export function getPaginationParams({
         return paginationBase;
     }
 
+    // When using multiple sort options, the API expects an array of sort options and the
```

Comment:
Asking for understanding, not necessarily suggesting, whether you can imagine any benefit in type inference if TypeScript declaration of the function associates multiple sort options in argument with multiple sort options in return?

My impression is not, but the special case for client sorting of policies brought to mind.

Reply:
Absolutely yes. Initially I aimed for type inference where if it was detected that `sortFields` contained a nested array it would infer the return type, function args, etc. as the multiple version, but realized I needed to still support the single option case even if multiple were provided due to backwards compatibility with user saved URLs.

It might still be possible to infer the single case when only single case options are provided, just to simplify the API of the most common use case. I can investigate when time, possibly as a follow up depending on the level of complexity.